### DISCUSSION TITLE
Futures that should be added

### DISCUSSION THREAD

**Discussion by ZGRxSELO** [deepjyoti30/ytmdl-web-v2#73] [Ideas]
I believe this is one of the best downloaders for music with metadata. I would like to see the support of playlists and maybe an APK package so I can share the link directly for YouTube, just QoL things! Keep up the good work <3

**Comment by deepjyoti30** [comment]
Playlist support has been something that I have considered for a long time but due to un-availability of time, I haven't been able to work on it.

APK support is probably not going to happen anytime in the future because that will require some setting up to do.

Glad you like it btw!
